<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcontact</class>
 <widget class="QDialog" name="addcontact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_dodaj">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_odustani">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Odustani</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>171</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="imeLabel">
      <property name="text">
       <string>Ime</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="prezimeLabel">
      <property name="text">
       <string>Prezime</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ime"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_prezime"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_email"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="telefonLabel">
      <property name="text">
       <string>Telefon</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_telefon"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="kategorijaLabel">
      <property name="text">
       <string>Kategorija</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_kategorija">
      <item>
       <property name="text">
        <string>Biznis</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prijatelji</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rodbina</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dating</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="rodjendanLabel">
      <property name="text">
       <string>Datum roÄ‘enja</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QDateEdit" name="dateEdit_rodjendan">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
